client/src/components/Donation/utils.ts

```diff
@@ -0,0 +1,10 @@
+const numToCommas = (num: number) =>
+  num.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
```

Comment:
I would consider using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat for this. It should automagically format according to the locale.

Reply:
Thank you for the suggestion. I have updated the function accordingly.